antenna-testing/antenna-frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testProjects/ExampleTestProject.java

```diff
@@ -87,15 +88,26 @@ public String getExpectedToolConfigurationProductVersion() {
 
     @Override
     public List<WorkflowStep> getExpectedToolConfigurationProcessors() {
-        final List<WorkflowStep> processors = new BasicConfiguration().getProcessors();
-        processors.stream()
-                .filter(s -> "Source Validator".equals(s.getName()))
-                .forEach( s -> {
-                    final Map<String, String> map = s.getConfiguration().getAsMap();
-                    map.put("missingSourcesSeverity","FAIL");
-                    s.setConfiguration(StepConfiguration.fromMap(map));
-                    s.setDeactivated(true);
-                });
+        final List<WorkflowStep> processors = new BasicConfiguration().getProcessors()
+                .stream()
+                .map( s -> {
+                    if(!"Source Validator".equals(s.getName())) {
+                        return s;
+                    }
+                    final Map<String, String> map = Stream.of(s.getConfiguration().getAsMap(), Collections.singletonMap("missingSourcesSeverity", "FAIL"))
+                            .flatMap(m -> m.entrySet().stream())
+                            .collect(Collectors.toMap(
+                                    Map.Entry::getKey,
+                                    Map.Entry::getValue,
+                                    (v1, v2) -> v2));
```

Comment:
Part of this could be extracted into its own method and maybe even put in the Abstract base class or some Utils class (reuse for code below).

However, I don't think it's necessary.